.github/workflows/trivy-scan.yml

```diff
@@ -0,0 +1,88 @@
+name: Trivy CVE Dependency Scanner
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  scan-all:
+    runs-on: ubuntu-latest
+    permissions:
+      security-events: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v1
+        with:
+          go-version: "1.17"
+      - name: Build the kapp-controller artifacts
+        run: |
+          ./hack/install-deps.sh
+          ./hack/build.sh
+
+          # docker image
+          docker build -t docker.io/carvel/kapp-controller:${{ github.sha }} .
+      - name: Install Trivy
+        run: |
+          brew install aquasecurity/trivy/trivy
+
+          # download the sarif format template
+          git clone --depth 1 https://github.com/aquasecurity/trivy
+      - name: Run Trivy Reports
```

Comment:
Can any of this be captured in the trivy config so there is less scripting?

Reply:
Hmm, I am not sure! Its a good shout, maybe we can reduce this down with some config file. 👍

Reply:
Trivy doesn't have any sort of config / defaults. However they do have some ENV_VARs you can use. I made a new commit that extracts some of the duplication to variables. I think it makes it a fair bit clear 👍